### ISSUE TITLE
MPDF Issue with php7.2

### ISSUE THREAD

**Issue by LordRahl90** [niklasravnsborg/laravel-pdf#102]
The code and the error are attached as images.

It worked before on another configuration; but this is a docker laravel setup and its returning the error.
<img width="905" alt="screen shot 2018-07-17 at 10 41 02 am" src="https://user-images.githubusercontent.com/5837848/42809735-31bb3f34-89ae-11e8-9d58-4208a3200398.png">
<img width="1664" alt="screen shot 2018-07-17 at 10 41 19 am" src="https://user-images.githubusercontent.com/5837848/42809737-32545232-89ae-11e8-9aa7-91b904a5f559.png">

**Comment by marufmax** [comment]
You are not passing an Array or Object to the count function. There's nothing to do with mPDF or PHP version.

**Comment by LordRahl90** [comment]
But i never had a reason to use that function. The function resides in the vendor folder. I didnt create that class.

**Comment by niklasravnsborg** [comment]
@LordRahl90 Yes, this seems to be related to https://github.com/mpdf/mpdf/issues/616 which is an issue with mPDF. Just to investigate this further: Can you ensure that the code executes properly if you use php 7.1 or 7.0?

**Comment by erfansahaf** [comment]
@niklasravnsborg We are using laravel-pdf on Production with `php 7.1` and everything is fine. I think this problem occurs when you are on `php 7.2`.

**Comment by niklasravnsborg** [comment]
@erfansahaf We are using php-7.2 on production without any problems so far.
